src/WebSocketServer/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "PluginConfiguration": {
+    "Servers": [
+      {
+        "Network": 860833102,
+        "BindAddress": "127.0.0.1",
+        "Port": 10334,
+        "SslCert": "",
+        "SslCertPassword": "",
+        "TrustedAuthorities": [],
+        "MaxConcurrentConnections": 40,
+        "SessionEnabled": false,
+        "SessionExpirationTime": 60
```

Comment:
Allowed clients?
Maybe it will be useful to provide a list of allowed users that can connect. Or do you believe that the load will be reasonable and won't lock node?

Reply:
to that i dont have a proper answer, will need well tested.